ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/video.js

```diff
@@ -0,0 +1,61 @@
+// LOGIC: Extract the & UUID of the youtube video and render it in iframe
+const videoCss = () => {
+	return [
+		{
+			inline: true,
+			mime: 'text/css',
+			text: `
+                .video-container {
+                    position: relative;
+                    width: 100%;
+                    height: 0;
+                    padding-bottom: 56.25%;
+                }
+                .video-iframe {
```

Comment:
Please prefix the class names with "jop" eg `.jop-video-container`. This is because notes can contain arbitrary HTML, especially those imported from web clipper, and so we add a prefix to avoid any conflict.